### ISSUE TITLE
Explore Data Page: Sort Buttons Issues (visibility, a11y)

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1054]
- [x] Explore relationships between x and y in z	SR	[A11Y - Health Equity Tracker]: Screen reader announces hidden content in Explore relationships page_Web_1.16(c)	"ENVIRONMENT:
macOS 11.3 | Chrome Version 90.0.4430.212 (Official Build) (x86_64) | Voiceover
Chrome OS Version 91.0.4472.60 (Official Build) beta (64-bit) | Chromevox
PRE REQUISITES:
Turn on the respective screen reader
STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/exploredata
2). Perform action on next button to reach the final sub page
3). Verify the options are selected as such, the page is 'Explore relationships between Diabetes and Covid-19 in United States.
4). Navigate to 'Diabetes cases per 100K' & 'Total Covid-19 cases per 100K' section on the page
5). Navigate through the section
6). Observe the announcement.
EXPECTED RESULT:
Screen reader should announce content available on the page and it shouldn't announce hidden content.
OBSERVED RESULT:
Screen reader announces content not visible on the page. Content under the Highest/Lowest rates is announced by screen reader even when the section is in collapsed state.
Replication vide attached"		Same issue is observed on Compare rates page.	

- [x] Explore relationships between x and y in z	SR	[A11Y - Health Equity Tracker]: Sort button on table column header doesn't have a label_Web_1.1	"ENVIRONMENT:
macOS 11.3 | Chrome Version 90.0.4430.212 (Official Build) (x86_64) | Voiceover
Chrome OS Version 91.0.4472.60 (Official Build) beta (64-bit) | Chromevox
PRE REQUISITES:
Turn on the respective screen reader
STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/exploredata
2). Perform action on next button to reach the final sub page
3). Verify the options are selected as such, the page is 'Explore relationships between Diabetes and Covid-19 in United States.
4). Navigate to 'Diabetes cases by Race & Ethnicity' & 'Total Covid-19 cases by Race & Ethnicity' section on the page
5). Navigate through the table under the section
6). Observe the announcement.
EXPECTED RESULT:
Button should a label and it should be properly announced by screen reader.
OBSERVED RESULT:
Sort button on table column header doesn't have a label. Screen reader announces the sort button only as button.
Replication video attached"	https://drive.google.com/file/d/1bieRiaR6NvleRlem692b8qpgKVQOI6Cr/view?usp=sharing	"Same issue is observed on Compare rates page.
Issue isreplicable in the following Android environment:
Device: Pixel 4
Android Version: 11
Chrome Browser Version: 91.0.4472.101

- [x] Explore relationships between x and y in z	SR	[A11Y - Health Equity Tracker]: Improper focus and announcement when user performs action on sort button_Web_1.3	"ENVIRONMENT:
macOS 11.3 | Chrome Version 90.0.4430.212 (Official Build) (x86_64) | Voiceover
Chrome OS Version 91.0.4472.60 (Official Build) beta (64-bit) | Chromevox
PRE REQUISITES:
Turn on the respective screen reader
STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/exploredata
2). Perform action on next button to reach the final sub page
3). Verify the options are selected as such, the page is 'Explore relationships between Diabetes and Covid-19 in United States.
4). Navigate to 'Diabetes cases by Race & Ethnicity' & 'Total Covid-19 cases by Race & Ethnicity' section on the page
5). Navigate to the sort button on table column header and activate
6). Observe the behavior & announcement.
EXPECTED RESULT:
Screen reader focus shouldn't reset when user performs action on sort button and the screen reader should announce the successful completion of sort in a proper manner. 
OBSERVED RESULT:
When user performs action on sort button using chromevox, screen reader focus resets to the top of the page. User has to renavigate the whole page content to reach the table which can be cumbersome for the user. When user performs action on sort button using voiceover, screen reader announces dimmed, which can be confusing for the user.
Replication video attached"	https://drive.google.com/file/d/1CUBGgkCCQ1omPi8brISJM_-W_QkgVrET/view?usp=sharing	Same issue is observed on Compare rates page.

**Comment by benhammondmusic** [comment]
https://user-images.githubusercontent.com/41567007/129074786-5d5dba20-7626-4daf-93f2-5b5d8fa86892.mov
